edi_purchase_oca/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "EDI Purchase",
+    "summary": """
+        Define EDI Configuration for Purchase Orders""",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": ["purchase", "edi_oca", "component_event"],
```

Comment:
@sebalix @LoisRForgeFlow I'm wondering why pulling the big edi_oca module as the code inside this does not depend on it ?

Reply:
that's wrong, see the model :wink: 
```
    _inherit = ["purchase.order", "edi.exchange.consumer.mixin"]
```

Reply:
Ouuuh, ok.